tevm/evm.go

```diff
@@ -120,8 +120,8 @@ type State struct {
 
 	Blocks Tree // key = n2h(blocknum) = hash, value = serialized block
 
-	logs      []*types.Log
-	snapshots []statesnap
+	Logs      []*types.Log
+	Snapshots []statesnap
```

Comment:
the export of an unexported field here is 🤢

Reply:
Yeah, it's gross.